### ISSUE TITLE
Reply directly from notification 

### ISSUE THREAD

**Issue by drew5494** [kelyvin/Android-Messages-For-Desktop#5]
Just another feature that would be cool!
![reply-message-notification-mac](https://user-images.githubusercontent.com/9094103/42051023-b1cba25c-7ad8-11e8-929a-c4c0113fb20e.jpg)

**Comment by fonzie24** [comment]
Definitely would be an awesome addition!

**Comment by LeapingDolphin23** [comment]
Agreed, although this could be hard to implement as it would require more than just a web wrapper so that the application can inject the text
